### DISCUSSION TITLE
Is it possible to install two ArgoCD on one Kubernetes cluster in different namespace with helm, argocd chart version 5.27.3?

### DISCUSSION THREAD

**Discussion by TomH54321** [argoproj/argo-cd#14117] [General]
the first one installation was fine in namespace argocd, but the second one was fail
older version argocd helm chart was fine to install two argocd in different namespace

helm install test-argocd2 . -n test-test-argocd2 --create-namespace

Error: INSTALLATION FAILED: rendered manifests contain a resource that already exists. Unable to continue with install: CustomResourceDefinition "applications.argoproj.io" in namespace "" ex
ists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.helm.sh/release-name" must equal "test-argocd2": current value is "argo
cd"; annotation validation error: key "meta.helm.sh/release-namespace" must equal "test-argocd2": current value is "argocd"

**Comment by jannfis** [comment]
It's possible that the [Argo Helm community](https://github.com/argoproj/argo-helm) would know more about this topic.

**Reply by TomH54321** [reply]
I try to insall second argocd in other namespace with below command:
helm install test-argocd2 . -n test-test-argocd2 --create-namespace

show below error
Error: INSTALLATION FAILED: rendered manifests contain a resource that already exists. Unable to continue with install: CustomResourceDefinition "applications.argoproj.io" in namespace "" ex
ists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.helm.sh/release-name" must equal "test-argocd2": current value is "argo
cd"; annotation validation error: key "meta.helm.sh/release-namespace" must equal "test-argocd2": current value is "argocd"

after adding below setting, was fine:
crds:
    install: false
    keep: true
    
I try older version argocd helm chart doesn't have this issue
